zenoh/__init__.pyi

```diff
@@ -948,11 +948,30 @@ class ZenohId:
     def into_keyexpr(self) -> KeyExpr: ...
     def __str__(self) -> str: ...
 
-def init_logging(basic_config: bool = True, **kwargs):
+@overload
+def init_logging(*, basic_config: bool = True, raw: Literal[False] = False, **kwargs):
+    """Start redirecting all zenoh logs to Python logger named `zenoh`.
+
+    If `basic_config=True`, which is the default, it also calls
+    `logging.basicConfig(**kwargs)` as a convenience.
+
+    If the overloaded signature `init_logging(raw=True)` is called, then
+    zenoh logs are directly printed to stdout, bypassing the Python logger
+    (and its formatting). Raw log level can be controlled using `RUST_LOG`
+    environment variable.
+    """
+
+@overload
+def init_logging(*, raw: Literal[True]):
```

Comment:
An all bindings the function is called `try_init_log_from_env`. No `init_logging` function is provided. I suggest to keep consistency and to update bindings once a decision is taken (e.g. see https://github.com/eclipse-zenoh/zenoh/pull/1165).

Reply:
This function doesn't have the same behavior than `try_init_log_from_env`, and doesn't use it internally.
Do you suggest to replace it by a raw `try_init_log_from_env` wrapper? 

I would be ok with that. I just don't want to loose my code, so I can just put a `#[allow(unusued)]` for the moment.